project3/index.html

```diff
@@ -0,0 +1,232 @@
+            <span> <input id="real-name" type="text">
+            </span>
+
+            <label for="alias">Alias</label>
+            <span> <input id="alias" type="text">
+            </span>
+
+            <label for="email">Email</label>
+            <span> <input id="email" type="email">
+            </span>
+        </fieldset>
+
+
+        <fieldset class="personalcharacteristics-left-column">
+            <h2> Personal Characteristics</h2>
+            <label for="age">Age</label>
+            <span><input id="age" type="number"></span>
+
+            <span>
+                <legend>Category</legend>
+                <p class="categoryinfo">Please select as many as apply to your origin and how you gained your powers
+                </p>
+            </span>
+
+            <div class="options">
+                <input id="alien" type="checkbox">
+                <label for="alien">Alien</label>
+            </div>
+            <span>
+                <input id="cyborg" type="checkbox">
+                <label for="cyborg">Cyborg</label>
+            </span>
+            <span>
+                <input id="experiment" type="checkbox">
+                <label for="experiment">Experiment</label>
+            </span>
+            <span>
+                <input id="human" type="checkbox">
+                <label for="human">Human</label>
+            </span>
+            <span>
+                <input id="inhuman" type="checkbox">
+                <label for="inhuman">Inhuman</label>
+            </span>
+            <span>
+                <input id="mutant" type="checkbox">
+                <label for="mutant">Mutant</label>
+            </span>
+        </fieldset>
+
+        <fieldset class="personalcharacteristics-right-column">
+
+            <legend>Gender</legend>
+
+            <span>
+                <input name="gender" id="female" type="radio">
+                <label for="female">Female</label>
+            </span>
+            <span>
+                <input name="gender" id="male" type="radio">
+                <label for="male">Male</label>
+            </span>
+            <span>
+                <input name="gender" id="nonbinary" type="radio">
+                <label for="nonbinary">Nonbinary</label>
+            </span>
+            <span>
+                <input name="gender" id="queer-fluid" type="radio">
+                <label for="queer-fluid">Genderqueer/Genderfluid</label>
+            </span>
+            <span>
+                <input name="gender" id="no-answer" type="radio">
+                <label for="no-answer">Prefer not to answer</label>
+            </span>
+            <span>
+                <input name="gender" id="self-describe" type="radio">
+                <label for="self-describe">Prefer to self describe</label>
+            </span>
+            <span>
+                <textarea class="textarea" name="" id="self-describe" cols="30" rows="5"></textarea>
```

Comment:
This `textarea` can't have the same `id` as the `input` above; `id`s must be unique.